src/main/java/wellnus/reflection/feature/ReflectionManager.java

```diff
@@ -32,9 +33,16 @@ public class ReflectionManager extends Manager {
     private static final String FAV_COMMAND = "fav";
     private static final String PREV_COMMAND = "prev";
     private static final String NO_ELEMENT_MESSAGE = "There is no new line of input, please key in inputs!";
-    private static final String INVALID_COMMAND_MESSAGE = "The command is invalid!!";
-    private static final String INVALID_COMMAND_NOTES = "Please try 'help' command to check the "
-            + "available commands and their usages!";
+    private static final String INVALID_COMMAND_MESSAGE = "Invalid command issued!";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String INVALID_COMMAND_NOTES =
+            "get command " + GetCommand.COMMAND_USAGE + LINE_SEPARATOR
+            + "like command " + LikeCommand.COMMAND_USAGE + LINE_SEPARATOR
+            + "unlike command " + UnlikeCommand.COMMAND_USAGE + LINE_SEPARATOR
+            + "fav command " + FavoriteCommand.COMMAND_USAGE + LINE_SEPARATOR
+            + "prev command " + PrevCommand.COMMAND_USAGE + LINE_SEPARATOR
+            + "home command " + HomeCommand.COMMAND_USAGE + LINE_SEPARATOR
+            + "exit command " + ExitCommand.COMMAND_USAGE;
```

Comment:
Similarly, I think you should modify this to something like:
```
Supported commands in 'reflect':
<list commands>
Try 'help [command-to-check]' to learn more about a specific command!
```

Reply:
Also, on line 175 I think you should log the `BadCommandException` message, i.e. do `LOGGER.log(Level.INFO, badCommand.getMessage())` instead since the new usage note message is super long and floods the log, example:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/9844161/230107893-6f2aa9fb-d6e3-42a0-b121-a62f265ab326.png">
As much as log will be redirected to a file later, we still should avoid flooding the log which would make it less readable